From 9b72aa7cdbc2a81cffc6f855933afe90c81046d5 Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@arm.com>
Date: Wed, 28 Feb 2024 12:40:34 +0000
Subject: [PATCH] Skip aggregator test

This test case is known to be flaky upstream[1] and often fails on the
autobuilder[2], so skip it until this has been resolved upstream.

[1] https://gitlab.freedesktop.org/gstreamer/gstreamer/-/issues/410
[2] https://bugzilla.yoctoproject.org/show_bug.cgi?id=15054

Upstream-Status: Inappropriate
Signed-off-by: Ross Burton <ross.burton@arm.com>
---
 tests/check/libs/aggregator.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tests/check/libs/aggregator.c b/tests/check/libs/aggregator.c
index 1f2c5b4..27b3ac7 100644
--- a/tests/check/libs/aggregator.c
+++ b/tests/check/libs/aggregator.c
@@ -1475,7 +1475,9 @@ gst_aggregator_suite (void)
   tcase_add_test (general, test_flushing_seek);
   tcase_add_test (general, test_infinite_seek);
   tcase_add_test (general, test_infinite_seek_50_src);
-  tcase_add_test (general, test_infinite_seek_50_src_live);
+  // This test case is known to be flaky, remove it until resolved:
+  // https://gitlab.freedesktop.org/gstreamer/gstreamer/-/issues/410
+  // tcase_add_test (general, test_infinite_seek_50_src_live);
   tcase_add_test (general, test_linear_pipeline);
   tcase_add_test (general, test_two_src_pipeline);
   tcase_add_test (general, test_timeout_pipeline);
-- 
2.34.1

